### PR TITLE
Creating Modal component + Adding X Icon to icons folder

### PR DESCRIPTION
**Pull request by RamboGj**

### CODE REVIEW

File: src/components/icons/X.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react"
```
Comment (elliotBraem) [inline src/components/icons/X.jsx:1]: Should this be in this PR?

File: apps/builddao/widget/components/Modal.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
+const { theme } = VM.require("rambo-dev.near/widget/ThemeProvider");
```
Comment (elliotBraem) [inline apps/builddao/widget/components/Modal.jsx:1]: Can we not use the ThemeProvider please? We can explore this in the future.

Theming does not need to happen on a per-component basis. We will just define the Theme in the root.

Comment (RamboGj) [reply apps/builddao/widget/components/Modal.jsx:1]: Totally

File: src/components/icons/X.jsx (hunk 1 above)
Comment (RamboGj) [reply src/components/icons/X.jsx]: Removed

**Pull request merged**